test/binMaxHeadTest.html

```diff
@@ -178,6 +205,30 @@
     }
 
 var s = null;
+
+describe('Automated Test', function() {
+            it('binMaxHead(s, x, w)', function() {
```

Comment:
Add parenthesizations of the prosodic trees (x, y, z, a) to the display text of the it() statements, and also the expected violations.